src/handlers/registration.py

```
@@ -34,20 +34,20 @@ async def cb_reg_start(c: CallbackQuery, state: FSMContext):
             return
         
         await state.clear()
-        await delete_all_bot_messages(c.from_user.id)
+        await delete_all_bot_messages(c.from_user.id, bot_instance)
         await state.set_state(RegistrationStates.nick)
-        text = f"1/3 · {T['progress'][0]}"
+        text = f"1/3 - {T['progress'][0]}"
         
         try:
             msg = await c.message.edit_text(text, reply_markup=kb_wizard(0))
         except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)